alt_src/alt_src.py

```diff
@@ -2145,6 +2145,10 @@ def setup_logging(options):
     handler = logging.StreamHandler(sys.stdout)
     handler.setFormatter(logging.Formatter(options.config['log_format']))
     handler.setLevel(output_log_level)
+
+    # remove the default handler created by logging.basicConfig in main function
```

Comment:
Hmm... sorry, I'm having a doubt that this makes sense (or at least the comment).

`logger` here is the "altsrc" logger.

`logging.basicConfig` is expected to install a handler on the *root* logger.

So, wouldn't logger.handlers already be empty here?  Why would altsrc logger have handlers at all, at this point in the code?

Also: with the current approach then, this is still only fixing the setup correctly for altsrc logger.  Messages from other loggers won't end up going to the stream or file handlers here since it's only being installed on the altsrc logger.

This is what I said in an earlier comment:

> I would say the desired setup (which alt-src actually doesn't implement correctly right now) is:
> 
> - only the root logger has handlers
> - it always has exactly one StreamHandler
> - it may also have an additional FileHandler (if log_file is set)
> 

I think that's still valid and it'd be preferable to implement it that way.